server/src/main/java/org/elasticsearch/inference/InferenceService.java

```diff
@@ -78,8 +78,8 @@ default void init(Client client) {}
      * Whether this service should be hidden from the API. Should be used for services
      * that are not ready to be used.
      */
-    default Boolean hideFromConfigurationApi() {
-        return Boolean.FALSE;
+    default boolean hideFromConfigurationApi() {
```

Comment:
Some refactoring, I think we can use a primitive here since I don't believe we'll ever want to return null.